components/brave_vpn/brave_vpn_utils.cc

```diff
@@ -5,13 +5,39 @@
 
 #include "brave/components/brave_vpn/brave_vpn_utils.h"
 
+#include "base/command_line.h"
 #include "base/feature_list.h"
 #include "brave/components/brave_vpn/features.h"
+#include "brave/components/brave_vpn/switches.h"
+#include "brave/components/brave_vpn/url_constants.h"
 
 namespace brave_vpn {
 
 bool IsBraveVPNEnabled() {
   return base::FeatureList::IsEnabled(brave_vpn::features::kBraveVPN);
 }
 
+std::string GetManageUrl() {
```

Comment:
Nice cleanup here (moving to utils) 😄 I might rework this code a bit in the future and use it myself for the credentials code (since we need to use the same environment). In my case, I just need to know the environment (not the manage URL)